config/index.js

```diff
@@ -75,57 +160,129 @@ module.exports = {
     'no-proto': 'error',
+    'no-prototype-builtins': 'error',
     'no-redeclare': 'error',
     'no-regex-spaces': 'error',
     'no-restricted-globals': ['error', 'event'],
+    'no-restricted-imports': 'off',
+    'no-restricted-modules': 'off',
+    'no-restricted-properties': 'off',
+    'no-restricted-syntax': 'off',
     'no-return-assign': ['error', 'except-parens'],
+    'no-return-await': 'error',
+    'no-script-url': 'error',
     'no-self-assign': 'error',
     'no-self-compare': 'error',
     'no-sequences': 'error',
+    'no-setter-return': 'error',
+    'no-shadow': 'error',
     'no-shadow-restricted-names': 'error',
     'no-spaced-func': 'error',
     'no-sparse-arrays': 'error',
+    'no-sync': 'off',
+    'no-tabs': 'error',
+    'no-template-curly-in-string': 'error',
+    'no-ternary': 'off',
     'no-this-before-super': 'error',
     'no-throw-literal': 'error',
     'no-trailing-spaces': 'error',
     'no-undef': 'error',
     'no-undef-init': 'error',
+    'no-undefined': 'off',
+    'no-underscore-dangle': 'off',
     'no-unexpected-multiline': 'error',
     'no-unmodified-loop-condition': 'error',
     'no-unneeded-ternary': ['error', { 'defaultAssignment': false }],
     'no-unreachable': 'error',
     'no-unsafe-finally': 'error',
+    'no-unsafe-negation': 'error',
     'no-unused-expressions': ['error', { 'allowShortCircuit': true, 'allowTernary': true }],
+    'no-unused-labels': 'error',
     'no-unused-vars': ['error', { 'vars': 'all', 'args': 'all', 'argsIgnorePattern': '[_]+' }],
     'no-use-before-define': ['error', { 'functions': false }],
     'no-useless-call': 'error',
+    'no-useless-catch': 'error',
     'no-useless-computed-key': 'error',
+    'no-useless-concat': 'error',
     'no-useless-constructor': 'error',
     'no-useless-escape': 'error',
+    'no-useless-rename': 'error',
+    'no-useless-return': 'error',
     'no-var': 'error',
+    'no-void': 'error',
+    'no-warning-comments': 'off',
     'no-whitespace-before-property': 'error',
     'no-with': 'error',
+    'nonblock-statement-body-position': 'error',
+    'object-curly-newline': ['error', {
+      'consistent': true,
+      'multiline': true,
+    }],
     'object-curly-spacing': ['error', 'always'],
+    'object-property-newline': 'off',
+    'object-shorthand': 'error',
     'one-var': ['error', { 'initialized': 'never' }],
+    'one-var-declaration-per-line': 'off',
+    'operator-assignment': 'error',
     'operator-linebreak': ['error', 'after', { 'overrides': { '?': 'ignore', ':': 'ignore' } }],
     'padded-blocks': 'off',
+    'padding-line-between-statements': [
+      'error',
+      { blankLine: 'always', prev: 'directive', next: '*' },
+      { blankLine: 'any', prev: 'directive', next: 'directive' },
+    ],
+    'prefer-arrow-callback': 'off',
     'prefer-const': 'error',
+    'prefer-destructuring': 'error',
+    'prefer-exponentiation-operator': 'off',
+    'prefer-named-capture-group': 'error',
```

Comment:
Named capture groups are pretty awesome, but technically we're targeting ES2017 now, and this is in ES2018. Moreover, these are still not supported in Firefox (see: https://kangax.github.io/compat-table/es2016plus/#test-RegExp_named_capture_groups), and we aren't currently using a Babel transformation for this either.

Reply:
Good catch, updated